### PR TITLE
Fixes #11049: Create a command to display system classes

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/11049

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

File: share/commands/agent-info [hunk 1]
```diff
@@ -3,9 +3,24 @@
...
+# @man *-v*: run the agent in verbose mode, prints detailed information
+
 
 . "${BASEDIR}/../lib/common.sh"
 
+VERBOSE=false
+
+while getopts "vd" opt; do
+  case $opt in
+    v|d)
```
Comment (peckpeck) [inline share/commands/agent-info]: Why support -d ? It is not documented

Comment (amousset) [reply share/commands/agent-info]: indeed

**Comment by amousset** [comment]
Commit modified

**Pull request merged**